src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/MorganStanley.ComposeUI.DesktopAgent.csproj

```diff
@@ -4,8 +4,8 @@
 		<ImplicitUsings>enable</ImplicitUsings>
 		<Nullable>enable</Nullable>
 		<LangVersion>preview</LangVersion>
-		<AssemblyName>MorganStanley.ComposeUI.Fdc3.$(MSBuildProjectName)</AssemblyName>
```

Comment:
Can you please explain this change?

Reply:
The name this way was MorganStanley.ComposeUI.Fdc3.MorganStanley.ComposeUI.DesktopAgent